src/components/users/ResetPasswordForm.js

```diff
@@ -0,0 +1,128 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Loader from 'react-loader-spinner';
+import { object, string } from 'prop-types';
+
+import { userActions } from '../../actions/user.actions';
+import '../../style/App.scss';
+import './user-form.scss';
+import FormInput from '../common/FormInput';
+import { userRequest } from '../../actions/user.actions';
+
+const ResetPasswordForm = ({ intl, url_uid, url_token }) => {
```

Comment:
camelCase here too
```suggestion
const ResetPasswordForm = ({ intl, urlUid, urlToken }) => {
```

Reply:
Also I think in other PR I mentioned you can use `useIntl` to avoid prop drilling with intl

Reply:
Yes, I will remove intl from every component in other PR